filament/src/components/RenderableManager.h

```diff
@@ -64,6 +64,12 @@ class FRenderableManager : public RenderableManager {
 
     static_assert(sizeof(Visibility) == sizeof(uint16_t), "Visibility should be 16 bits");
 
+    struct MorphTargets {
+        FMorphTargetBuffer* buffer = nullptr;
+        size_t offset = 0;
+        size_t count = 0;
```

Comment:
I think we can use `uint32_t` here, this way this structure would become 16 bytes instead of 24.

Reply:
Ok it's done.